### ISSUE TITLE
I am getting error when using ngmodel

### ISSUE THREAD

**Issue by sharmilemurugaraj** [morocarlo/ng2-my-datepicker#1]
Hi,
  I have configured this module and when I run my site, I am getting the following error:

```
   ORIGINAL EXCEPTION: No value accessor for '', Can you tell me where I am going wrong?
```

**Comment by morocarlo** [comment]
Hi, i have update the readme with a "Use Example". Could you provide plnkr demo with your error?

**Comment by apavillet** [comment]
Getting the same error @sharmilemurugaraj did you find the reason?

**Comment by amarewgs** [comment]
same here,

**Comment by sharmilemurugaraj** [comment]
yes, since we are using ngmodel, two binding is not working. Try upgrading the angular form version and check.

**Comment by Taats** [comment]
yep, doesnt work.

**Comment by amarewgs** [comment]
I have the latest angular forms version but keeps me throwing Error: Uncaught (in promise): No value accessor for 'fromDate'. I think the lib is not using the latest FORMS_DIRECTIVES found in the common package

**Comment by Taats** [comment]
got it to working using the following imports:

import {Component, ViewContainerRef, Input, Output, EventEmitter, AfterViewInit, Provider, forwardRef } from '@angular/core';
import {NgModel, FORM_DIRECTIVES, ControlValueAccessor} from '@angular/forms';
import {NgIf, NgFor, NgClass, NG_VALUE_ACCESSOR } from '@angular/common';

and add these lines of code:

const CUSTOM_VALUE_ACCESSOR = new Provider(NG_VALUE_ACCESSOR, { useExisting: forwardRef(() => DatePicker), multi: true });

@Component({ // add:
providers: [CUSTOM_VALUE_ACCESSOR],
}
